server/cluster/cluster.go

```diff
@@ -161,6 +163,146 @@ func (c *Cluster) UpdateShardTables(ctx context.Context, shardTablesArr []ShardT
 	return nil
 }
 
+// OpenTable will open an existing table on the specified shard.
+// The table to be opened must have been created.
+func (c *Cluster) OpenTable(ctx context.Context, request OpenTableRequest) error {
```

Comment:
It is necessary to check whether it is a partition table, and the normal table cannot open table.